packages/xy-chart/src/series/PointSeries.jsx

```diff
@@ -1,18 +1,17 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-
 import { Group } from '@vx/group';
-import { GlyphDot } from '@vx/glyph';
 import { chartTheme, color } from '@data-ui/theme';
 
 import { callOrValue, isDefined } from '../utils/chartUtils';
 import { pointSeriesDataShape } from '../utils/propShapes';
+import GlyphDotComponent from '../glyph/GlyphDotComponent';
 
 export const propTypes = {
   data: pointSeriesDataShape.isRequired,
   label: PropTypes.string.isRequired,
   labelComponent: PropTypes.element,
-
+  pointComponent: PropTypes.oneOfType([PropTypes.func, PropTypes.element]),
```

Comment:
should this just be element since we're calling `React.createElement`?

Reply:
I tried just using element, but I got the warning that the passed component was a func.. not an element.